src/main/drivers/accgyro/accgyro_spi_mpu6500.c

```diff
@@ -83,6 +83,20 @@ uint8_t mpu6500SpiDetect(const busDevice_t *bus)
     return mpuDetected;
 }
 
+//read temperature for 6500
+static bool mpu6500ReadTemperature(gyroDev_t *gyro, int16_t *tempData)
```

Comment:
This will have to be tested with all of the non - MPU6500 gyros to make sure it doesn't cause problems.